app/components/Poll/index.js

```diff
@@ -9,247 +9,225 @@ import { Link } from 'react-router-dom';
-                      <td className={styles.graphColumn}>
-                        {votes === 0 ? (
-                          <span className={styles.noVotes}>Ingen stemmer</span>
-                        ) : (
-                          <div className={styles.fullGraph}>
-                            <div
-                              style={{
-                                width: `${ratio}%`,
-                              }}
-                            >
-                              <div className={styles.pollGraph}>
-                                {ratio >= 18 && <span>{`${ratio}%`}</span>}
-                              </div>
-                            </div>
-                            {ratio < 18 && (
-                              <span style={{ marginLeft: '2px' }}>
-                                {`${ratio}%`}
-                              </span>
-                            )}
-                          </div>
-                        )}
-                      </td>
-                    </tr>
-                  );
-                })}
-              </tbody>
-            </table>
-            {resultsHidden && (
-              <p style={{ fontStyle: 'italic', marginTop: 15 }}>
-                Resultatet er skjult for vanlige brukere.
-              </p>
+        <Link to={`/polls/${id}`}>
+          <Flex
+            justifyContent="center"
+            alignItems="center"
+            className={styles.headerBar}
+          >
+            {!details && description.length !== 0 ? (
+              <Tooltip content="Trykk for mer info" renderDirection="right">
+                {title}
+              </Tooltip>
+            ) : (
+              <> {title} </>
             )}
           </Flex>
-        )}
-        {!hasAnswered && (
-          <Flex column className={styles.optionWrapper}>
-            {!expanded && (
-              <Flex
-                className={styles.blurContainer}
-                onClick={this.toggleTruncate}
-              >
-                <p className={styles.blurOverlay}>
-                  Klikk her for å se alle alternativene.
-                </p>
-                <Icon
-                  className={cx(styles.blurOverlay, styles.blurArrow)}
-                  size={60}
-                  name={expanded ? 'arrow-up' : 'arrow-down'}
-                />
-              </Flex>
-            )}
-            {options &&
-              optionsToShow.map((option) => (
-                <Flex
-                  className={cx(
-                    styles.alignItems,
-                    expanded ? '' : styles.blurEffect
-                  )}
-                  key={option.id}
-                >
+        </Link>
+      </Flex>
+      <Flex
+        column
+        alignItems="center"
+        className={styles.contentWrapper}
+        style={{
+          height:
+            isExpanded && optionsRef.current
+              ? `${optionsRef.current.clientHeight}px`
+              : '0',
```

Comment:
I guess this is kind of a design opinion, but i think it looks better if the opening transition doesn't play while loading default-expanded polls. This is the code I used to make it not do that.

```suggestion
          height: isExpanded
            ? optionsRef.current
              ? `${optionsRef.current.clientHeight}px`
              : 'auto'
            : 0,
```

Reply:
I love small moving things, but i realize it's not ideal for [everyone using our plattform](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion). I'll remove it for now :3